.github/workflows/package.yml

```diff
@@ -0,0 +1,99 @@
+name: Package Application
+
+on:
+  pull_request:
+    branches:
+      - '*'
+  push:
+    tags:
+      - 'v*'
+
+jobs:
+  release:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [macos-latest, ubuntu-latest, windows-latest]
+    steps:
+      - name: Retrieves project from the git branch
+        uses: actions/checkout@v2
+
+      - name: Read .nvmrc
+        run: echo "##[set-output name=NVMRC;]$(cat package.json | grep '\"node\":' | sed 's/^ *//;s/ *$//;s/\"node\":\ \"//;s/\",//' )"
+        id: nvm
+
+      - name: Setup node ${{ steps.nvm.outputs.NVMRC }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: '${{ steps.nvm.outputs.NVMRC }}'
+
+      - run: node --version; npm --version; yarn --version
+
+      - name: Cache node modules
+        id: cache
+        uses: actions/cache@v2
+        with:
+          path: node_modules
+          key: ${{ runner.OS }}-build-${{ hashFiles('**/yarn.lock') }}
+
+      - name: Install Dependencies
+        if: steps.cache.outputs.cache-hit != 'true'
+        run: yarn install
+
+      - name: Add MacOS certificates
+        if: matrix.os == 'macos-latest'
+        run: |
+          KEYCHAIN=build.keychain
+          security create-keychain -p $MACOS_KEYCHAIN_PASSWORD $KEYCHAIN
+          security default-keychain -s $KEYCHAIN
+          security unlock-keychain -p $MACOS_KEYCHAIN_PASSWORD $KEYCHAIN
+          security import <(echo $CERTIFICATE_OSX_APPLICATION | base64 --decode) \
+                          -f pkcs12 \
+                          -k $KEYCHAIN \
+                          -P $CERTIFICATE_PASSWORD \
+                          -T /usr/bin/codesign
+          security set-key-partition-list -S apple-tool:,apple: -s -k $MACOS_KEYCHAIN_PASSWORD $KEYCHAIN
+        env:
+          CERTIFICATE_OSX_APPLICATION: ${{ secrets.CERTIFICATE_OSX_APPLICATION }}
+          CERTIFICATE_PASSWORD: ${{ secrets.CERTIFICATE_PASSWORD }}
+          MACOS_KEYCHAIN_PASSWORD: ${{ secrets.MACOS_KEYCHAIN_PASSWORD }}
+
+      # To add when we have a Windows signing certificate
+      # - name: Add Windows certificate
+      #   if: matrix.os == 'windows-latest' && startsWith(github.ref, 'refs/tags/')
+      #   id: write_file
+      #   uses: timheuer/base64-to-file@v1
+      #   with:
+      #     fileName: 'win-certificate.pfx'
+      #     encodedString: ${{ secrets.CERTIFICATE_WINDOWS_PFX }}
+
+      - name: Make artifact (${{ matrix.os }})
```

Comment:
This gives the following warning: `You have set packagerConfig.ignore, the Electron Forge webpack plugin normally sets this automatically. Your packaged app may be larger than expected if you dont ignore everything other than the '.webpack' folder`

I don't see changes to this though, but I haven't gotten that error locally.

Reply:
Yeah it showed up when I moved the forge config outside of the package.json. Will look into that